src/Api/todos.tsx

```diff
@@ -0,0 +1,21 @@
+import { NewTodo, Todo } from '../types/Todo';
+import { client } from '../utils/fetchclient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const deleteTodo = (id: number) => {
+  return client.delete(`/todos/${id}`);
+};
+
+export const postTodo = (todo: NewTodo) => {
+  return client.post<Todo>('/todos', todo);
+};
+
+export const updateTodo = (
```

Comment:
Consider adding a comment above the `updateTodo` function to explain its purpose.